### ISSUE TITLE
overrideSpecBasePath not working

### ISSUE THREAD

**Issue by piglab** [Axway-API-Management-Plus/apim-cli#395]
### APIM-CLI version

1.13.5

### API-Management version

7.7.0

### Bug description

In my api-config.json file, I added the parameter backendBasepath

When I run api import, with the option -overrideSpecBasePath an a openApi v2 swagger, the host parameter is not overrided.

It works fine with open API 3 servers parameter.

Is this a bug ?

### Steps to reproduce

import an API with a swagger OpenAPI v2
```

{
    "swagger": "2.0",
    "info": {
        "title": "M/TEXT Integration Adapter Rest API",
        "description": "M/TEXT Integration Adapter Rest API",
        "version": "6.11",
        "contact": {
            "name": "kühn & wey Software GmbH",
            "url": "http://www.kwsoft.de",
            "email": "info@kwsoft.de"
        },
        "license": {
            "name": "Commercial license",
            "url": "http://www.kwsoft.de"
        }
    },
    "tags": [
        {
            "name": "mtext-integration-adapter"
        }
    ],
    "host": "REPLACEME",
    "schemes": [
        "https"
    ],
    "basePath": "/mtext-integration-adapter",
    "produces": [
        "application/json"
    ],
```

### Relevant log output

```shell
No error in console
```

**Comment by piglab** [comment]
```
{
    "name" : "{{tpl_api_name}}",
    "path" : "{{tpl_api_path}}",
    "state" : "{{tpl_api_state}}",
    "version" : "{{tpl_api_version}}",
    "organization" : "{{tpl_api_organization}}",
    "summary" : "{{tpl_api_summary}}",
    "image" : "{{tpl_api_image}}",
    "descriptionType" : "manual",
    "descriptionManual" : "{{tpl_api_description}}",
    "tags" : {
      "deployment" : [ "AzureDevops" ],
      "deploymentDate" : [ "{{tpl_api_timestamp}}" ]
    },
    "backendBasepath" : "{{tpl_api_backendBasepath}}",
....

```

**Comment by lpaquet79** [comment]
Hi,

I have the same problem with the **overrideSpecBasePath** option.

This works well for OAS 3.0 but with **Swagger 2** the Backend service URL is replaced by the swagger one.

Whereas in the case of OAS 3.0, the Backend service URL is replaced by the backendBasepath.

It should be the same behavior

**Comment by rathnapandi** [comment]
Hi @piglab  and @lpaquet79 
Looks like a duplicate of #397

**Comment by rathnapandi** [comment]
Hi @piglab, overridebasepath is not handled for swagger 2.  We will fix it as part of next release.

**Comment by piglab** [comment]
Great ! Thank you

**Comment by rathnapandi** [comment]
Hi @piglab & @lpaquet79 , 
For swagger 2, host and basepath are defined in two separate places, we are just updating basepath not host name and protocol / scheme as those values can be overridden via frontend api .   

For open api, the url is a single entity and we are replacing the whole value with backendbasepath.

**Comment by rathnapandi** [comment]
fixed in v1.14.1
